Misc/NEWS.d/next/Library/2024-11-19-10-46-57.gh-issue-124130.OZ_vR5.rst

```diff
@@ -0,0 +1,2 @@
+Fix a bug in matching regular expression ``\B`` in empty string. Now it
```

Comment:
I think this change requires to be mentioned in a What's New entry. This is a nontrivial change and could be burried in the huge changelog.